Palaso/Settings/CrossPlatformSettingsProvider.cs

```diff
@@ -14,17 +14,27 @@ namespace Palaso.Settings
 	/// </summary>
 	public class CrossPlatformSettingsProvider : SettingsProvider, IApplicationSettingsProvider
 	{
+		//Public only for unit testing. I can't get InternalsVisibleTo to work, possibly because of strong naming.
+		public const string UserConfigFileName = "user.config";
```

Comment:
I tried InternalsVisibleTo and cannot figure out why it doesn't work. Probably something to do with strong naming. Eberhard's idea should, though.